### DISCUSSION TITLE
How to detect the black background of the page page?

### DISCUSSION THREAD

**Discussion by LastMonody** [pymupdf/PyMuPDF#1481] [Looking for help]
Hello, I am currently doing a pdf parsing task, but we seem to have no problem in my pdf, but I use page.drawings() to draw the image in the original image to the new page page, but the new one The page generates a black background that completely blocks the text on the page. Is there any way to parse the page when only the layer that we see with the naked eye is parsed?

**Comment by JorjMcKie** [comment]
I am confused by your description of the problem - please try again.

**Reply by LastMonody** [reply]
Thanks for your reply, I have solved it!
